_episodes/03-running-containers.md

```diff
@@ -222,28 +232,26 @@ test.txt
 ~~~
 {: .output}
 
-So this shows us that we can exit Docker containers for arbitrary lengths of time and then
+So this shows us that we can exit containers for arbitrary lengths of time and then
 return to our working environment inside of them as desired.
 
 >## Clean up a container
 >
->If you want a container to be [cleaned up][docker-docs-run-clean-up] &mdash; that is
+>If you want a container to be [cleaned up][podman-docks-run-clean-up] &mdash; that is
 >deleted &mdash; after you exit it then run with the `--rm` option flag
 >
 >~~~bash
->docker run --rm -it <IMAGE> /bin/bash
+>podman run --rm -it <IMAGE> /bin/bash
 >~~~
 >{: .source}
 {: .callout}
 
-[docker-docs-run]: https://docs.docker.com/engine/reference/run/
-[docker-docs-run-detached]: https://docs.docker.com/engine/reference/run/#detached-vs-foreground
-[docker-docs-run-clean-up]: https://docs.docker.com/engine/reference/run/#clean-up---rm
+[podman-docks-run]: https://docs.podman.io/en/stable/markdown/podman-run.1.html
 [docker-hub-python]: https://github.com/docker-library/python
-[docker-docs-ps]: https://docs.docker.com/engine/reference/commandline/ps/
-[docker-docs-rename]: https://docs.docker.com/engine/reference/commandline/rename/
-[docker-docs-start]: https://docs.docker.com/engine/reference/commandline/start/
-[docker-docs-attach]: https://docs.docker.com/engine/reference/commandline/attach/
-[docker-docs-exec]: https://docs.docker.com/engine/reference/commandline/exec/
+[podman-docks-ps]: https://docs.podman.io/en/stable/markdown/podman-ps.1.html
+[podman-docks-rename]: https://docs.docker.com/engine/reference/commandline/rename/
+[podman-docks-start]: https://docs.docker.com/engine/reference/commandline/start/
+[podman-docks-attach]: https://docs.docker.com/engine/reference/commandline/attach/
+[podman-docks-exec]: https://docs.docker.com/engine/reference/commandline/exec/
```

Comment:
not a big deal, but why are these aliased to "podman-docks" instead of "podman-docs"?

Reply:
Typo when using the replace command :D